images/plugin-compiler/Dockerfile

```diff
@@ -20,9 +18,9 @@ COPY images/plugin-compiler/data/build.sh /build.sh
 # RUN cd $TYK_GW_PATH && go mod tidy && go mod vendor
 RUN chmod +x /build.sh
 
-RUN cd $TYK_GW_PATH &&  \
-    git clone --depth 1 --branch $TYK_GW_TAG  https://github.com/TykTechnologies/tyk . && \
-    go mod vendor
+COPY src.tgz .
```

Comment:
go.mod is already in `extra_files`, is this not sufficient to create a vendor folder?

Reply:
No, it doesn't seem to work. With just the go.mod, the vendor command doesn't just fetch any vendor dependencies.